index.html

```diff
@@ -121,6 +131,28 @@ <h5>About Myself</h5>
             </ul>
         </div>
     </section>
+
+    <footer class="footer">
+        <div class="footerhouse">
+            <section class="footer-section">
+                <div class="footercontainer">
+                    <h6>Contact me</h6>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a
+                        project
+                        that needs coding. I'd love to help with it!</p>
+                </div>
+            </section>
+            <form action="https://formspree.io/f/xlekzbpj" action="post" class="form">
+                <ul>
+                    <li><input class="input" type="text" placeholder="Lorem Ipsum" maxlength="30" required></li>
+                    <li><input class="input" type="email" placeholder="loremipsum@gmail.com" required></li>
+                    <li><textarea class="input" name="user_message" id="message" cols="30" rows="6" maxlength="500"
+                            required>Write your message here...</textarea></li>
+                    <li><button class="footer-btn" type="submit">Get in touch</button></li>
```

Comment:
- Please implement the hover interaction style for your form button as you have done with the other buttons in your project. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Reply:
@mckent05 Thank you  will make the suggested changes here as well 🙏🏾